.github/workflows/build-apps.yaml

```diff
@@ -2,25 +2,15 @@ name: build apps
 
 on:
   push:
-    branches:
-      - '**'
+    paths:
+      - 'digiwf-apps/**'
```

Comment:
Das müsste wieder für alle sein.
Wir hatten explizit alle Jobs auf allen Pfaden aktiv, da man damit diese Jobs als Required einstellen kann. Über den Pfad werden die Jobs nur bei Änderungen an den jeweiligen Dateien aktiv. Diese Unterscheidung kann man bei dem Required nicht machen

Reply:
Meiner Meinung nach nicht, deswegen ist ja beim MR das `synchronize` dazu gekommen. Das zählt ja nur bei einem normalen Push.
Bei einem MR wird immer alles gebaut.

Reply:
Meine Anforderung aus dem obigen Grund ist, dass bei jedem PR, egal welche Änderungen wo gemacht wurden, immer alles gebaut wird (Apps, Maven, Docs). Wenn das der Fall ist, dann passt es für mich.

Reply:
Siehe diesen MR, da wurde ja auch alles gebaut, obwohl es keinem Filter entspricht

Reply:
Was mich verwundert hat, waren die Laufzeiten. Bei anderen MRs ist das viel länger.

![image](https://github.com/it-at-m/digiwf-core/assets/10849776/3edccc67-fa79-474d-92d0-2d39c94c4489)